src/core/GUIDReferenceHandler.ttslua

```diff
@@ -0,0 +1,100 @@
+local GuidReferences = {
+  White = {
+    ClueCounter = "d86b7c",
+    ClickableClueCounter = "db85d6",
+    DamageCounter = "eb08d6",
+    HandZone = "a70eee",
+    HorrorCounter = "468e88",
+    InvestigatorSkillTracker = "e598c2",
+    Playermat = "8b081b",
+    ResourceCounter = "4406f0",
+    Trash = "147e80"
+  },
+  Orange = {
+    ClueCounter = "1769ed",
+    ClickableClueCounter = "3f22e5",
+    DamageCounter = "e64eec",
+    HandZone = "5fe087",
+    HorrorCounter = "0257d9",
+    InvestigatorSkillTracker = "b4a5f7",
+    Playermat = "bd0ff4",
+    ResourceCounter = "816d84",
+    Trash = "f7b6c8"
+  },
+  Green = {
+    ClueCounter = "032300",
+    ClickableClueCounter = "891403",
+    DamageCounter = "1f5a0a",
+    HandZone = "0285cc",
+    HorrorCounter = "7b5729",
+    InvestigatorSkillTracker = "af7ed7",
+    Playermat = "383d8b",
+    ResourceCounter = "cd15ac",
+    Trash = "5f896a"
+  },
+  Red = {
+    ClueCounter = "37be78",
+    ClickableClueCounter = "4111de",
+    DamageCounter = "591a45",
+    HandZone = "be2f17",
+    HorrorCounter = "beb964",
+    InvestigatorSkillTracker = "e74881",
+    Playermat = "0840d5",
+    ResourceCounter = "a4b60d",
+    Trash = "4b8594"
+  },
+  Mythos = {
+    AllCardsBag = "15bb07",
+    BlessCurseManager = "5933fb",
+    CampaignThePathToCarcosa = "aca04c",
+    DataHelper = "708279",
+    DeckImporter = "a28140",
+    DoomCounter = "85c4c6",
+    DoomInPlayCounter = "652ff3",
+    InvestigatorCounter = "f182ee",
+    MasterClueCounter = "4a3aa4",
+    MythosArea = "9f334f",
+    NavigationOverlayHandler = "797ede",
+    OptionPanelSource = "830bd0",
+    PlayArea = "721ba2",
+    PlayAreaZone = "a2f932",
+    PlayerCardPanel = "2d30ee",
+    ResourceTokenBag = "9fadf9",
+    RulesReference = "d99993",
+    SoundCube = "3c988f",
+    TokenArranger = "022907",
+    TokenSource = "124381",
+    TokenSpawnTracker = "e3ffc9",
+    TourStarter = "0e5aa8",
+    Trash = "70b9f6",
+    VictoryDisplay = "6ccd6d"
+  }
+}
+
+function getObjectByOwnerAndType(param)
+  local owner = param.owner
+  local type = param.type
+  return getObjectFromGUID(GuidReferences[owner][type])
+end
+
+function getObjectsByType(type)
+  local objList = {}
+  for owner, objects in pairs(GuidReferences) do
+    local obj = getObjectFromGUID(objects[type])
+    if obj then
+      objList[owner] = obj
+    end
+  end
+  return objList
+end
```

Comment:
I'm a little concerned about just how intuitive it will be to use this for the 'Mythos' owner, since arguably at least half of those aren't really applicable at all, it's just a catch-all.  Would it make sense to maybe refactor the `getObjectsByType` method to return a lone object instead of a list if only 1 is found, or maybe create a new method specifically for these singletons?